tests/Support/TestingShipment.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Tests\Support;
+
+use Closure;
+use Exception;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
+use GuzzleHttp\Psr7\Response;
+use MelhorEnvio\MelhorEnvioSdkPhp\Shipment;
+
+class TestingShipment extends Shipment
+{
+    private HandlerStack $handlerStack;
+    private array $recorded;
```

Comment:
Acha que esse nome deixaria mais claro do que se trata ? Ou $recordedRequests

```suggestion
    private array $requestHistory;
```

Reply:
Esse eu copiei o padrão que o Laravel utiliza no [cliente HTTP deles](https://laravel.com/docs/9.x/http-client) :v

Acho que vale a pena utilizar esse só pra facilitar.

O que acha?

Reply:
Eu entendo que por ser utilizado assim no laravel ficaria mais "familiar", mas ainda acho que talvez ficasse mais claro da maneira que sugeri, até porque não sei se é exatamente uma função muito utilizada desse http client do laravel, então não sei se a "familiaridade" iria funcionar tanto, mas se realmente prefere fazer assim por mim não tem problema, ainda dá pra inferir do que se trata o atributo